### ISSUE TITLE
[usda-branch] - 508- fix color contrast of top right menu

### ISSUE THREAD

**Issue by Atticus29** [Symbiota/Symbiota#1044]
The top right menu links fail proper color contrast. Refer to the screenshot below.

![image005](https://github.com/BioKIC/Symbiota/assets/2775448/c957718a-8109-4838-bed7-0a7519eb67ea)

**Comment by Atticus29** [comment]
Fixed in https://github.com/BioKIC/Symbiota/commit/27db78a5ecbff79b8844bc4b61cf06d626b3ab5f.
